modules/overviews/app/components/overviews/project_custom_fields/items_component.html.erb

```diff
@@ -0,0 +1,16 @@
+<%=
+  flex_layout do |flex|
+    @project_custom_fields.each_with_index do |project_custom_field, i|
+      margin = i == @project_custom_fields.size - 1 ? 0 : 3
+      flex.with_row(mb: margin) do
```

Comment:
Too bad we can't use something like `gap` here. That would make things easier.

Reply:
@cbliard we can use gap. We'll just have to implement it as an inline style for now. @oliverguenther did the same in a0ab3a0c.